index.bs

```diff
@@ -2534,6 +2549,49 @@ create them does not matter.
  1. Return « |branch1|, |branch2| ».
 </div>
 
+<div algorithm>
+ <dfn abstract-op lt="ReadableStreamFromIterable" id="readable-stream-from-iterable">
+ ReadableStreamFromIterable(|asyncIterable|)</dfn> performs the following steps:
```

Comment:
Editorial nit: this has only one call site, so it might be best to inline it?

Otherwise, it'd be good to alphabetize it within the abstract ops section.

Reply:
I prefer it as an abstract op, in case we ever want to export it for other specifications. Besides, all of the other `ReadableStream` methods also have corresponding abstract ops.

I'll fix the alphabetical order though.